PHPDaemon/Servers/WebSocket/Protocols/V13.php

```diff
@@ -29,6 +29,8 @@ class V13 extends Connection
 
     protected $framebuf = '';
 
+    protected $firstFrameOpcodeName;
```

Comment:
This property store _first_ frame opcode name when message is fragmented. So it is  $firstFrameOpcodeName. Why not ?